docs/async/async-overview.rst

```diff
@@ -0,0 +1,30 @@
+Async Overview
+==============
+
+Modern apps and services are expected to be responsive.
+
+* Client applications are expected to be always-on, always-connected, but keep the UI free for the user to interact with.
+* Services are expected to be able gracefully handle a heavy load.
+
+.NET provides the capability for apps and services to be responsive with easy-to-use, language-level asynchronous programming models in C#/VB and F#.  Although similar, the nuances of async programming in F# differs from that in C# and VB.
+
+Why Async?
+----------
+
+Here are a few reasons:
+
+* Responsiveness in apps and services is expected by users, and even slight UX hangups are often punished harshly (one-star reviews).
+* Almost all modern apps demand elements which block on I/O in some way.
```

Comment:
This feels incomplete. I would add a line that ties it into the first bullet, that is, the point is that because modern apps demand elements that block on I/O, it is easy to create a non-responsive UX, or aysnc is needed but usually complex.